draft-ietf-rats-architecture.md

```diff
@@ -316,6 +316,26 @@ Relying Party:
 : A remote server that will securely grant the Attester
 permission to continue operating (i.e., not reboot) for a period of time
 
+## FIDO Biometric Authentication
+
+In the FIDO protocol, it is the device in the user’s hand that authenticates the human user by biometrics like fingerprints or PIN and password.
+The comparison of the input from the user to the reference or expected values is done by the device, often a mobile phone, not the relying party.
+For the relying party to know that the authentication is trustworthy the relying party needs to know that the authenticator part of the device is trustworthy.
+The FIDO protocol employs attestation for this. 
+
+The FIDO protocol supports several attestation protocols and a mechanism by which new ones can be registered and added. Attestation defined by RATS is thus a candidate for use by FIDO. 
```

Comment:
meaning, for use by the FIDO protocol?  or for use by the FIDO Alliance?

Reply:
protocol; addressed in recent commit.